### PR TITLE
Release process

### PR DESCRIPTION
**Pull request by bilgili**

### CODE REVIEW

**Comment by bilgili** [comment]
It is still 0.5 !

File: doc/Changelog.md [hunk 1]
```diff
@@ -3,6 +3,14 @@ Changelog {#Changelog}
 
 # master {#master}
 
+# Release 0.5 (2016-04-06) {#Release050}
+
+* [#270](https://github.com/BlueBrain/Livre/pull/270):
+  Fix #113. SSE selection algorithm is simplified
+  Fix #255, #221 Regression on visible selection fixed
+  Default camera is position is modified
```
Comment (jafyvilla) [inline doc/Changelog.md]: remove first 'is'

**Comment by jafyvilla** [comment]
Change doesn't match the PR name

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by jafyvilla** [comment]
+1

**Pull request merged**